### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by saifudinbadani**
Screen Shot 2022-02-11 at 6 09 40 pm
Screen Shot 2022-02-11 at 6 09 33 pm

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks good to me

**Review by BelideNikhil — APPROVED** [review]
Try using variables for font-weight
Rest is good!

File: css/common.css [hunk 1]
```diff
@@ -82,6 +82,7 @@ body {
 .heading-1 {
   font-size: var(--tp1);
   line-height: var(--lh1);
```
Comment (nshanbhag20001412) [inline css/common.css:84]: color variable naming can be improved

**Review by nshanbhag20001412 — APPROVED** [review]
rest looks good to me

**Pull request merged**